stock_quant_package_dimension/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,2 @@
+This module adds dimension fields on stock packages and an estimated
+weight (in kg).
```

Comment:
I think you should add the file `USAGE.rst` to the description settings and workflow.